lib/config.js

```diff
@@ -0,0 +1,14 @@
+import nconf from 'nconf';
+
+// Load configuration
+// First look for overrides from the command line or environment variables
+nconf.argv().env();
+
+// Then look for overrides from modules that are importing arc-js
+const path = 'daostack.json'; // Path to configuration override file in calling module
+nconf.file("external", path);
```

Comment:
would this be better named "arc-js.config"?

Reply:
To what location in the application does 'daostack.json' resolve?  How can the application change this location?

Reply:
What if the application wants to supply config values that are computed at runtime, rather hardcoded on disk?  How is that done?

Reply:
im still really not sure what naming makes sense for anything. to me these parameters are more relevant to daostack than arc since they are more in reference to interaction with the blockchain.

Reply:
The location is the root of the project that imports that arc-js module. I was thinking of adding the ability pass in a different path to this file, just didnt get to it yet. let me think about the best way to support overriding config values at runtime. you can do it already since nconf makes it easy, you can just call nconf.overrides({ key: value}), but i will think about whether there a better interface for this

Reply:
The user shouldn't need to know that these settings pertain to arc, arc is encapsulated, and the settings may not *always* pertain to arc, they may also someday be specific to arc-js.

DAOstack is the name of the company, not the package.

Reply:
i'm still confused on naming, it seems weird to call both repos arc to me. and daostack is in the name of the package. we could split up the settings though into multiple sections though

Reply:
My understanding has been that the contracts are "Arc" and the JavaScript library is  "Arc-Js", each with "DAOstack-" prepended.

It's also my expectation that all of that could still be in flux...

I agree that the repos should have the same names.

Reply:
i renamed the file to arc.config.json

Reply:
I think this should be arc-js.config.json.

Reply:
hmm, so ugly. arc seems fine to me personally

Reply:
Yeah, the problem is that arc refers to a different package.  This one is named arc-js, and that is what we're configuring here.  Better to be ugly than to be confusing or misleading, IMHO.

Reply:
well sort of, i really think of this file as configuring both arc and arc-js, together. they work hand in hand. i could see arc.config.json being used to configure the arc repo too.